app/templates/public/index.hbs

```diff
@@ -3,35 +3,37 @@
     {{sanitize model.event.description}}
   </div>
   <div class="ui hidden divider"></div>
-  <div class="tickets" id="tickets">
-    <div class="ui segments ticket-list">
-      <div class="ui segment">
-        <h3 class="ui header">{{t 'Tickets'}}</h3>
-      </div>
-      <div class="ui segment">
-        {{#if model.event.isTicketingEnabled}}
-          {{public/ticket-list ticketUrl=model.event.ticketUrl isTicketingEnabled=model.event.isTicketingEnabled tickets=model.tickets}}
-        {{else}}
-          <div class="ui grid">
-            <div class="ui row">
-              <div class="column">
-                <a target="_blank" rel="noopener noreferrer" href="{{ticketUrl}}" class="ui right labeled blue icon button">
-                  <i class="ticket icon"></i>
-                  {{t 'Order tickets'}}
-                </a>
+  {{#if (or ticketUrl (gt model.tickets.length 0))}}
```

Comment:
You can just do `{{#if (or ticketUrl model.tickets)}}` length of 0 is considered to be _falsy_ by handlebars